src/js/App/Sidenav/SideNav.scss

```
@@ -0,0 +1,11 @@
+.ins-c-app-title {
+    color: white;
+    padding: calc(8px + var(--pf-global--spacer--sm)) 12px;
+    width: 100%;
+    margin-top: 7px;
+    border-bottom: 3px solid #3c3f42;
```

Can you change this to use PF css vars?
```suggestion
    border-bottom: var(--pf-global--spacer--xs) solid var(--pf-global--Color--300);
```